src/main/java/uk/gov/hmcts/reform/professionalapi/service/impl/OrganisationServiceImpl.java

```diff
@@ -149,7 +149,7 @@ private void addDxAddressToContactInformation(List<DxAddressCreationRequest> dxA
     public OrganisationsDetailResponse retrieveOrganisations() {
         List<Organisation> organisations = organisationRepository.findAll();
         log.debug("Received new organisation details...");
```

Comment:
Please change logger to indicate retrieveOrganisation retrieval flow

Reply:
I have included these changes in my latest commit. Please review. Thanks!